tests/Makefile

```diff
@@ -31,6 +31,7 @@ ROOT_DIR:=$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
 BIN_DIR := $(ROOT_DIR)/bin
 TOP_DIR := $(ROOT_DIR)
 LIBWEBSOCKET_LIB_DIR = $(ROOT_DIR)/../framework/libwebsockets-4.3.3/build/lib
+CURL_LIB_DIR = $(ROOT_DIR)/../framework/curl/curl-master/build/lib/
```

Comment:
don't we need to copy the library to the lib directory in the normal build case also, not just the test?

Reply:
Yes, that needs to be done in ut-core.